integration/src/test/scala/akka/persistence/jdbc/integration/MySQLScalaJdbcDurableStateChangesByTagTest.scala

```diff
@@ -3,10 +3,14 @@ package akka.persistence.jdbc.integration
 import com.typesafe.config.ConfigFactory
 import akka.actor.ActorSystem
 import akka.persistence.jdbc.state.scaladsl.JdbcDurableStateSpec
-import akka.persistence.jdbc.testkit.internal.Mysql
+import akka.persistence.jdbc.testkit.internal.MySQL
+import org.scalatest.Ignore
 
+// FIXME this test doesn't pass because of something with SequenceNextValUpdater
```

Comment:
Issue for this?

Reply:
I pinged the contributor in in the PR where he just added this the other day